src/BlockTypes/ProductGalleryThumbnails.php

```diff
@@ -39,6 +39,19 @@ protected function get_block_type_uses_context() {
 	 * @return string Rendered block type output.
 	 */
 	protected function render( $attributes, $content, $block ) {
+		$post_id = isset( $block->context['postId'] ) ? $block->context['postId'] : '';
```

Comment:
I believe we can enhance the code's conciseness by using the null coalescing operator (??). i.e.
```
$post_id = $block->context['postId'] ?? '';
```